src/javascripts/app.js

```diff
@@ -0,0 +1,17 @@
+var React = require('react');
+
+
+// Here is two empty lines, baddddd
+class Test {
+  constructor() {
+    this.testSuite = [];
+  }
+
+  supportES6() {
+    return "Does Hound support ES6 syntax?" ;
+  }
+}
+
+export default Test;
```

Comment:
Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.<br>Expected '(end)' and instead saw 'default'.